radboudumc-theme/sass/_typography.scss

```diff
@@ -33,11 +55,64 @@ h1.display-title {
   margin-bottom: 1em;
 }
 
-
 // First header on each page
 .page-title {
   font-size: 3rem;
+  font-weight: 800;
+
+  margin-bottom: 1.5rem;
+  font-size: 2rem;
+}
+
+p a { 
+  font-weight: 800;
+}
+
+h3 a {
   font-weight: 300;
+}
 
-  margin-bottom: 1em;
+div.container > p,
+div.page-content > p {
+  margin-bottom: 0.75em;
+}
+
+@media (min-width: 768px) {
+  div.container > p,
```

Comment:
What is the usage of '>' here? Should this be a strict child selector? I can imagine it has to apply for all elements within the content.

Reply:
There can be paragraphs in cards within the div.container, you do not want padding there, we only want them on direct children.

Reply:
You're right, didn't think of that. Let's keep it at this then!